spec/controllers/papers_controller_spec.rb

```diff
@@ -190,13 +190,13 @@
     it "should 404 for a paper that has been rejected" do
       rejected_paper = create(:paper, state: 'rejected')
       get :show, params: {id: rejected_paper.sha}, format: "html"
-      expect(response.status).to eq(404)
+      expect(response.status).to eq(302)
```

Comment:
I think this is OK. We could probably rewrite these tests to make them more comprehensive.